Code/Engine/Core/Input/Declarations.h

```diff
@@ -256,6 +256,7 @@ EZ_DECLARE_FLAGS_OPERATORS(ezInputSlotFlags);
 #define ezInputSlot_KeyRightAlt       "keyboard_right_alt"
 #define ezInputSlot_KeyLeftWin        "keyboard_left_win"
 #define ezInputSlot_KeyRightWin       "keyboard_right_win"
+#define ezInputSlot_KeyMenu           "keyboard_menu"
```

Comment:
Can't remember anymore, but maybe key_apps is meant to be the same.

Reply:
It indeed was the same.